lib/Data.js

```diff
@@ -30,22 +30,32 @@ export class Suite {
     }
 
     get runtime() {
-        return this.getAllTests()
+        var runtime = this.getAllTests()
             .map((x) => x.runtime)
             .reduce((sum, x) => sum + x, 0);
+        if (isNaN(runtime)) {
```

Comment:
If one of the tests  has not been run yet, its runtime is undefined. If we add undefined and a number we end up with NaN. We can check all tests before, but that's probably not easier?